daemon/hostdaemon.go

```diff
@@ -88,6 +107,43 @@ func (d *HostDaemon) connectWithRetry() {
 	d.client = pb.NewBridgePortServiceClient(conn)
 }
 
+func (d *HostDaemon) addHandler(req *cnitypes.PodRequest) (*cni100.Result, error) {
+	pf := 0
```

Comment:
We don't want magic numbers in the codebase. Is it possible to make these as constants?

Reply:
these are dummy values for testing and will be removed.
the actual values will come from the req.